### PR TITLE
Adds support for Jaeger allinone deployment strategy

### PR DESCRIPTION
**Pull request by rhomast**

### CODE REVIEW

**Comment by rhomast** [comment]
Is trace data stored in in-memory storage visible from the Jaeger console?

File: content/en/docs/observability/tracing/application-tracing.md [hunk 1]
```diff
@@ -32,14 +32,14 @@ In Verrazzano, Jaeger is comprised of the following components:
...
-- Storage, may be ephemeral or backed by OpenSearch.
+- Storage, which may be ephemeral (in-memory) or backed by OpenSearch.
    - Jaeger can be configured to use one of several choices for [storage back ends](https://www.jaegertracing.io/docs/{{<jaeger_doc_version>}}/deployment/#span-storage-backends).
    - In Verrazzano, trace records are stored in OpenSearch; OpenSearch must be enabled in Verrazzano for this to work.
 
-Jaeger is very configurable. See the Jaeger documentation for detailed information about Jaeger [features](https://www.jaegertracing.io/docs/{{<jaeger_doc_version>}}/features/) and [architecture](https://www.jaegertracing.io/docs/{{<jaeger_doc_version>}}/architecture). In Verrazzano, Jaeger does not include the Ingester service, Kafka, or Sparks jobs.
+Jaeger is very configurable. See the Jaeger documentation for detailed information about Jaeger [features](https://www.jaegertracing.io/docs/{{<jaeger_doc_version>}}/features/) and [architecture](https://www.jaegertracing.io/docs/{{<jaeger_doc_version>}}/architecture). In Verrazzano, Jaeger does not include the Ingester service, Kafka, or Sparks jobs automatically but can be customized to implement them.
```
Comment (rhomast) [inline content/en/docs/observability/tracing/application-tracing.md:39]: Does this need further changes?  configure-tracing.md says you can configure Jaeger to use Kafka but also says Verrazzano does not support Jaeger Spark dependencies.

File: content/en/docs/observability/tracing/application-tracing.md [hunk 2]
```diff
@@ -32,14 +32,14 @@ In Verrazzano, Jaeger is comprised of the following components:
...
 
 ### Next steps
 
 - Enable Jaeger and customize your Verrazzano Jaeger installation. See [configure tracing]({{< relref "/docs/observability/tracing/configure-tracing.md" >}}).
 - Configure your applications to [send traces to Jaeger]({{< relref "/docs/observability/tracing/capture-traces.md" >}}).
-- Then, use the Verrazzano Jaeger console to [view traces]({{< relref "/docs/observability/tracing/view-traces.md" >}}).
+- Then, use the Verrazzano Jaeger console to [view traces]({{< relref "/docs/observability/tracing/view-traces.md" >}}) stored in OpenSearch.
```
Comment (rhomast) [inline content/en/docs/observability/tracing/application-tracing.md:45]: Does the Jaeger console show traces from in-memory storage?

File: content/en/docs/observability/tracing/application-tracing.md [hunk 3]
```diff
@@ -32,14 +32,14 @@ In Verrazzano, Jaeger is comprised of the following components:
 
 - Collector service, which receives traces from clients, processes them, and stores them in a storage back end.
 - Query service, which exposes the APIs for retrieving traces from storage and hosts the Jaeger console for searching and analyzing traces.
-- Storage, may be ephemeral or backed by OpenSearch.
+- Storage, which may be ephemeral (in-memory) or backed by OpenSearch.
    - Jaeger can be configured to use one of several choices for [storage back ends](https://www.jaegertracing.io/docs/{{<jaeger_doc_version>}}/deployment/#span-storage-backends).
    - In Verrazzano, trace records are stored in OpenSearch; OpenSearch must be enabled in Verrazzano for this to work.
```
Comment (rhomast) [inline content/en/docs/observability/tracing/application-tracing.md:37]: Should this sentence be deleted?

File: content/en/docs/observability/tracing/configure-tracing.md [hunk 4]
```diff
@@ -13,8 +13,10 @@ This document describes how to enable Jaeger and customize your Jaeger installat
 
 To use Jaeger in Verrazzano, you must first enable the `jaegerOperator` component in your Verrazzano custom resource. Here is
 an example YAML file that enables the Jaeger Operator. Verrazzano installs the Jaeger Operator in the
-`verrazzano-monitoring` namespace. If OpenSearch and Keycloak components are enabled in the Verrazzano custom resource,
-then a default Jaeger instance is also created by the Jaeger Operator in the `verrazzano-monitoring` namespace.
+`verrazzano-monitoring` namespace. A default Jaeger instance is also created by the Jaeger Operator in the `verrazzano-monitoring` namespace.
```
Comment (rosemarymarano) [inline content/en/docs/observability/tracing/configure-tracing.md:16]: -> Also, a default Jaeger instance is created by the Jaeger Operator in the `verrazzano-monitoring` namespace.
(When written this way, it makes more sense to me. I'll explain if you want.)

File: content/en/docs/observability/tracing/configure-tracing.md [hunk 5]
```diff
@@ -13,8 +13,10 @@ This document describes how to enable Jaeger and customize your Jaeger installat
...
+
+If OpenSearch and Keycloak components are enabled in the Verrazzano custom resource, the Jaeger Operator implements the Production deployment strategy and stores its trace data using OpenSearch storage. If OpenSearch and Keycloak components are *not* enabled, the Jaeger Operator implements the AllInOne deployment strategy and stores its trace data using in-memory storage. Use the AllInOne strategy for development, testing, or demo purposes only. See [Jaeger Deployment Strategies](https://www.jaegertracing.io/docs/{{<jaeger_doc_version>}}/operator/#deployment-strategies) for more information.
```
Comment (rosemarymarano) [inline content/en/docs/observability/tracing/configure-tracing.md:18]: -> If ..., then... (I try to always use the If/then construct, if applicable) From the editing checklist:
Use of if/then construction for clarity and accurate translation: If you begin a sentence with if to introduce a condition, then insert then prior to the described consequence of the if clause to facilitate comprehension and translation accuracy by providing this transition. For example, If a certain condition is true, then a particular result happens.

-> For more information, see [Jaeger Deployment Strategies](https://www.jaegertracing.io/docs/{{<jaeger_doc_version>}}/operator/#deployment-strategies). 
(Another editing/stem sentence standard we have tried to follow, put the important part of the sentence at the end.)

File: content/en/docs/observability/tracing/configure-tracing.md [hunk 6]
```diff
@@ -64,9 +66,9 @@ If the Jaeger Operator component is enabled in the managed cluster, after succes
 a Jaeger collector service runs in the managed cluster, which exports the traces to the OpenSearch
 storage configured in the admin cluster.
 
-**NOTES**:
-   - Traces are exported to the admin cluster only when the Jaeger instance in the admin cluster is configured with OpenSearch storage.
-   - If the admin cluster is unavailable or the storage backend, OpenSearch, is not operational, then Jaeger will start dropping traces because it cannot buffer much data. To buffer large amounts of data, you would need to set up an intermediary queue, such as [Kafka](https://www.jaegertracing.io/docs/{{<jaeger_doc_version>}}/deployment/#kafka). This would prevent data loss from Jaeger collectors.
+If OpenSearch storage is not configured in the admin cluster, you will need to use Jaeger install overrides in the Verrazzano custom resource to explicitly enable and configure a Jaeger instance to export traces in the managed cluster to in-memory storage or to an external OpenSearch storage location.
```
Comment (rosemarymarano) [inline content/en/docs/observability/tracing/configure-tracing.md:69]: -> Jaeger to install overrides

**Review by rosemarymarano — COMMENTED** [review]
Just a few minor edits/suggestions

File: content/en/docs/observability/tracing/configure-tracing.md (hunk 6 above)
Comment (rhomast) [reply content/en/docs/observability/tracing/configure-tracing.md:69]: It's actually the 'install overrides' of 'jaeger'. Would you prefer 'Jaeger's install overrides'?

Comment (rosemarymarano) [reply content/en/docs/observability/tracing/configure-tracing.md:69]: OH. Hmmm, is this correct -> you will need to use Jaeger installation overrides?

File: content/en/docs/observability/tracing/application-tracing.md (hunk 1 above)
Comment (ayaranga) [reply content/en/docs/observability/tracing/application-tracing.md:39]: Yes, we can remove `Sparks jobs`

File: content/en/docs/observability/tracing/application-tracing.md (hunk 2 above)
Comment (ayaranga) [reply content/en/docs/observability/tracing/application-tracing.md:45]: Yes, Jaeger console shows traces when using in-memory storage as well

File: content/en/docs/observability/tracing/application-tracing.md (hunk 3 above)
Comment (ayaranga) [reply content/en/docs/observability/tracing/application-tracing.md:37]: Yes, thanks.

File: content/en/docs/observability/tracing/configure-tracing.md [hunk 7]
```diff
@@ -13,8 +13,10 @@ This document describes how to enable Jaeger and customize your Jaeger installat
...
 an example YAML file that enables the Jaeger Operator. Verrazzano installs the Jaeger Operator in the
-`verrazzano-monitoring` namespace. If OpenSearch and Keycloak components are enabled in the Verrazzano custom resource,
-then a default Jaeger instance is also created by the Jaeger Operator in the `verrazzano-monitoring` namespace.
+`verrazzano-monitoring` namespace. Also, a default Jaeger instance is created by the Jaeger Operator in the `verrazzano-monitoring` namespace.
+
+If OpenSearch and Keycloak components are enabled in the Verrazzano custom resource, then the Jaeger Operator implements the Production deployment strategy and stores its trace data using OpenSearch storage. If OpenSearch and Keycloak components are *not* enabled, then the Jaeger Operator implements the AllInOne deployment strategy and stores its trace data using in-memory storage. Use the AllInOne strategy for development, testing, or demo purposes only. For more information, see [Jaeger Deployment Strategies](https://www.jaegertracing.io/docs/{{<jaeger_doc_version>}}/operator/#deployment-strategies) .
+
```
Comment (ayaranga) [inline content/en/docs/observability/tracing/configure-tracing.md:19]: nit: Extra space at the end.

**Pull request merged**